CMakeLists.txt

```diff
@@ -1,31 +1,64 @@
-cmake_minimum_required(VERSION 3.12)
+cmake_minimum_required(VERSION 3.14)
 
 include(pico_sdk_import.cmake)
 
 project(picoprobe)
 
+# On the next line, retain "pico" for Raspberry Pi Pico,
+# or set 'PICO_BOARD' to a board name as defined in
+# https://github.com/raspberrypi/pico-sdk/tree/master/src/boards/include/boards
+#
+# For example,
+#    "adafruit_qtpy_rp2040"    for the Adafruit QTPy
+#    "adafruit_feather_rp2040" for the Adafruit Feather RP2040
+#    "sparkfun_promicro"       for the SparkFun Pro Micro RP2040
+
+set(PICO_BOARD "pico")
```

Comment:
Should this be defined in this file? A PICO_BOARD environment variable works too. Not having to modify any files make building even easier.

Linux/MacOS

```bash
export PICO_BOARD=SPARKFUN_PROMICRO
cmake .
```

WIndows
```batch
SET PICO_BOARD=SPARKFUN_PROMICRO
cmake .
```

PowerShell Core (All OS'es)
```powershell
$env:PICO_BOARD=SPARKFUN_PROMICRO
cmake .
```